NineChronicles.DataProvider/GraphQL/Types/NineChroniclesSummaryQuery.cs

```diff
@@ -63,14 +63,19 @@ public NineChroniclesSummaryQuery(MySqlStore store)
             Field<ListGraphType<EquipmentRankingType>>(
                 name: "EquipmentRanking",
                 arguments: new QueryArguments(
+                    new QueryArgument<StringGraphType> { Name = "avatarAddress" },
                     new QueryArgument<StringGraphType> { Name = "itemSubType" },
                     new QueryArgument<IntGraphType> { Name = "limit" }
                 ),
                 resolve: context =>
                 {
+                    string? address = context.GetArgument<string?>("avatarAddress", null);
+                    Address? avatarAddress = address == null
+                        ? (Address?)null
+                        : new Address(address[2..]);
```

Comment:
Is it okay we always slice 2 letters from the `address`?  IMO replacing `"0x"` to `""` is safer if want to strip `0x`